CHANGES.md

```diff
@@ -77,3 +77,7 @@ The following parameters have been removed from the DC/OS installer:
 * --install-prereqs
 * --deploy
 * --postflight
+
+### Fixed and improved
+
+* Fix preflight docker version check failing for docker 1.19. (DCOS-56831)
```

Comment:
Is the plan to "fix" this in 1.13.x?

Then this entry should not be here at all.

Reply:
> Is the plan to "fix" this in 1.13.x?
> Then this entry should not be here at all.

Why? if something is fixed in 1.13.x it does not imply it's fixed in 1.14. Bugfix release for 1.13 may be months after 1.14 so IMO we should include all fixes in changelog. Look at [Mesos Changelog](https://github.com/apache/mesos/blob/master/CHANGELOG) 1.8.0 release contains bugfixes that were also listed in [1.7.3 CHANGELOT](https://github.com/apache/mesos/blob/1.7.x/CHANGELOG)

Do we have a different policy for DCOS changelog? Were can I find information about it?

Reply:
> Why? if something is fixed in 1.13.x it does not imply it's fixed in 1.14.

That's right, but here we know already more, and we want to use that information: we _know_ that the next 1.13.x release will be much earlier than the GA 1.14.0 release. 

You are right that once 1.14.0 is already out the door and if we are then going to release 1.13.9 that this is a different situation. But that's not where we are yet.

> Do we have a different policy for DCOS changelog? Were can I find information about it?

https://github.com/dcos/dcos/wiki/CHANGES.md-guidelines#major-releases

Quote:

> In case of a bug fix forward port into the development version of the next major release there is no need to mention this in the changelog of the major release. Example: a bug is fixed for the upcoming 1.12.3 release while 1.13.0 is still in development. As of common sense the same bug fix gets forward-ported into 1.13.0-dev. That very forward-port does not need to be accompanied by a changelog entry. It's expected behavior for this bug fix to be included in 1.13.0. If it's not, it's a bug in itself and will be fixed with with a future 1.13.x release, accompanied with a relevant changelog entry.

I hope you appreciate that this
- saves you work (forward ports don't need to be noted during *all* of 1.14.0 dev, this saves a bunch of work).
- keeps the information in the 1.14.0 changelog more interesting (all the goodies from previous major / patch releases are contained, everyone assume that anyway, we don't need to mention them again).

I want to say again: in a couple of months from now you will be right. But not yet :)

Hope that helps @janisz